CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+[v1.1.7] - 2019-08-18
```

Comment:
Unreleased section should be above v1.1.7 section.

Also note that the Unreleased changes were added to v1.1.6 in bd43f270019cfbabe7a99b5002cfacea38fc6025, I hadn't pushed this change until just now.